automated/linux/docker-prune/disable_prune.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh -e
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+
+
+TOKEN=$(cat /etc/lmp-device-register-token)
+DEVICE_NAME=$(cat /etc/hostname)
+API_DATA=$(cat ./z-50-fioctl.toml)
+FACTORY=$(grep -w "LMP_FACTORY" /etc/os-release | cut -d'=' -f2 | sed 's/\"//g')
+APP="shellhttpd"
+
+check_prune () {
+    runtime="5 minute"
+    endtime=$(date -ud "$runtime" +%s)
+    FOUND=0
+
+    while [ ! "${FOUND}" -eq 1 ]; do
+        if [ "$(date -u +%s)" -ge "$endtime" ]; then
+            break
+            return 1
+        elif (journalctl --no-pager -u aktualizr-lite | grep "Pruning unused docker containers"); then
+            echo "Prune Successful"
+            FOUND=1
+            return  0
```

Comment:
do you need to set FOUND when you always return?
Can this be done in a simpler way?